.dep-versions

```diff
@@ -8,11 +8,11 @@ enzyme=v0.0.130
 
 # For a custom PL version, update the package version here and at
 # 'doc/requirements.txt
-pennylane=0.38.0.dev24
+
```

Comment:
Maybe it will be easier for people to add it back if we leave the key in place:
```suggestion
# pennylane=
```

Reply:
Done